src/locale/pa-in.js

```diff
@@ -1,41 +1,41 @@
 //! moment.js locale configuration
 //! locale : Punjabi (India) [pa-in]
-//! author : Harpreet Singh : https://github.com/harpreetkhalsagtbit
+//! author : Amandeep Singh : https://github.com/adsingh14
```

Comment:
@marwahaha 
I think this change should be reverted. As I am the original author of this file.